src/Employer.Web/Services/GetAssociatedEmployerAccountsService.cs

```diff
@@ -1,10 +1,49 @@
-﻿namespace Employer.Web.Services
+﻿using Microsoft.Extensions.Logging;
+using SFA.DAS.EAS.Account.Api.Client;
+using SFA.DAS.EAS.Account.Api.Types;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Esfa.Recruit.Employer.Web.Services
 {
     public class GetAssociatedEmployerAccountsService : IGetAssociatedEmployerAccountsService
     {
-        public string[] GetAssociatedAccounts(string userId)
+        private readonly ILogger<GetAssociatedEmployerAccountsService> _logger;
```

Comment:
Do we need the 'Get'. Would just AssociatedEmployerAccountsService suffice?

Reply:
This was because of the stub intention. Renamed it to `EmployerAccountService`